### ISSUE TITLE
Support `{LANG}` format variable in `lowerDetailsEditing` setting

### ISSUE THREAD

**Issue by 0xEAB** [iCrawl/discord-vscode#84]
Would it be possible to add support for the `{LANG}` format variable in the detail strings (e.g. `lowerDetailsEditing`)?

This would allow to display things like this:
```json
"discord.detailsEditing": "📝 Editing",
"discord.lowerDetailsEditing": "a {LANG} file",
```

**Comment by iCrawl** [comment]
Took some time to get to it, I added this in the commit mentioned above.

Released in version 2.12.0

**Comment by 0xEAB** [comment]
nice, thanks :ok_hand:
